README.md

```diff
@@ -1,3 +1,4 @@
 # utils
 
-Run `npm run dist` before commit!
+Execute `npm run lint-fix && npm run dist` before commit!
```

Comment:
What for? You ignore dist directory in `.npmignore` file.

Also maybe add `publish` script?

Reply:
.npmignore not contains `dist` directory!

Reply:
mixed up with `build`